internal/e2e/src/tenv.ts

```diff
@@ -890,13 +890,14 @@ function spawn(
   leftoverStack.use(proc);
 
   child.stdout.on('data', (x) => {
-    stdout += x.toString();
-    stdboth += x.toString();
+    const str = x.toString().trim();
+    stdout += str;
+    stdboth += str;
     pipeLog({ cwd, pipeLogs }, x);
   });
   child.stderr.on('data', (x) => {
     // prefer relative paths for logs consistency
-    const str = x.toString().replaceAll(__project, '');
+    const str = x.toString().trim().replaceAll(__project, '');
```

Comment:
Same question as above, why trim? New lines and spaces are a regular and valid occurrence in logs.

Reply:
Did for reasons such as Docker adds some weird characters. But now not needed imo